### ISSUE TITLE
[Feature Request]: Make an option to make `BrowserView` completely opaque

### ISSUE THREAD

**Issue by wheezard** [electron/electron#34329]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

Currently, when a `BrowserView` is on top of another `BrowserView`, the background view still displays its content over the foreground one, given that the FG browserview doesn't have a css-defined background color.

![Screenshot of a window with two BVs](https://user-images.githubusercontent.com/90904039/169857769-c1922f99-fbcf-45aa-822d-9b311827a969.png)
The blue is the background color of the foreground `BrowserView` (set via `view.setBackgroundColor()`), and the red - of the background view. The text is in the red view, but is displayed over the blue view, because the blue view has the background only set by `view.setBackgroundColor()` and not css.

### Proposed Solution

It would be nice to be able to have a `BrowserView.prototype.setOpaque(isOpaque: boolean)` function that will tell Electron to render the `BrowserView` over everything else (respecting its z-position on the window, of course).

An alternative way to solving this is just to render the background color of `BrowserView` above all the "lower" `BrowserViews`, covering them. But some people might actually rely on that functionality, so, perhaps, it's not the best solution.

Basically, with `view.setOpaque(true)`, it should look something like this:
![Overlapping Browser Views](https://user-images.githubusercontent.com/90904039/169860007-ffd67355-3da7-4a0d-a359-79c4efeacfc7.png)

### Alternatives Considered

Of course, the obvious solution is to set the `background-color: white`, or something non-transparent in the css of the `BrowserView` (that's how I made the second screenshot), however, `BrowserView`s are usually used to display external (and unknown) content, and you will either add `background-color: white` and affect non-transparent pages, or affect transparent pages by doing nothing.

**Comment by pushkin-** [comment]
Would [this PR](https://github.com/electron/electron/pull/33435) address this issue?

**Comment by wheezard** [comment]
Yes, thanks.

I don't know if I should close this now, becuase the PR hasn't appeared in a normal release yet.
